code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -57,54 +58,33 @@ Difficulty: Hard
 	desc = "You're not quite sure how a signal can be bloody."
 	invisibility = 100
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/adjustBruteLoss(amount, updating_health = TRUE, forced = FALSE)
-	. = ..()
-	if(. > 0 && prob(25))
-		var/obj/effect/decal/cleanable/blood/gibs/bubblegum/B = new /obj/effect/decal/cleanable/blood/gibs/bubblegum(loc)
-		if(prob(40))
-			step(B, pick(GLOB.cardinals))
-		else
-			B.setDir(pick(GLOB.cardinals))
-
-/obj/effect/decal/cleanable/blood/gibs/bubblegum
-	name = "thick blood"
-	desc = "Thick, splattered blood."
-	random_icon_states = list("gib3", "gib5", "gib6")
-	bloodiness = 20
-
-/obj/effect/decal/cleanable/blood/gibs/bubblegum/can_bloodcrawl_in()
-	return TRUE
-
 /mob/living/simple_animal/hostile/megafauna/bubblegum/Life()
 	..()
-	move_to_delay = CLAMP((health/maxHealth) * 10, 5, 10)
+	move_to_delay = CLAMP(round((health/maxHealth) * 10), 5, 10)
 
 /mob/living/simple_animal/hostile/megafauna/bubblegum/OpenFire()
-	anger_modifier = CLAMP(((maxHealth - health)/60),0,20)
+	anger_modifier = CLAMP(((maxHealth - health)/50),0,20)
 	if(charging)
 		return
 	ranged_cooldown = world.time + ranged_cooldown_time
 
-	var/warped = FALSE
-	if(!try_bloodattack())
+	blood_warp()
+
+	if(prob(25))
 		INVOKE_ASYNC(src, .proc/blood_spray)
-		warped = blood_warp()
-		if(warped && prob(100 - anger_modifier))
-			return
 
-	if(prob(90 - anger_modifier) || slaughterlings())
-		if(health > maxHealth * 0.5)
+	else if(prob(5+anger_modifier/2))
+		slaughterlings()
+	else
+		if(health > maxHealth/2 && !client)
 			INVOKE_ASYNC(src, .proc/charge)
 		else
-			if(prob(70) || warped)
-				INVOKE_ASYNC(src, .proc/charge, 2)
-			else
-				INVOKE_ASYNC(src, .proc/warp_charge)
+			INVOKE_ASYNC(src, .proc/triple_charge)
 
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/Initialize()
```

Comment:
Uh. I'm not sure a flat revert is what you want to go for, here.

Reply:
Whoops I changed some things back but missed that